lib/OpenQA/WebAPI/Controller/Running.pm

```diff
@@ -231,7 +221,18 @@ sub streaming {
             }
         });
 
-    $self->on(finish => sub { Mojo::IOLoop->remove($id) });
+    # tell worker to create live stream
+    OpenQA::Utils::log_debug('Telling worker to start providing livestream');
+    $ipc->websockets('ws_send', $worker->id, 'livelog_start', $job->id);
+
+    $self->on(
```

Comment:
Not sure, but might worth trying to use ```once``` here

Reply:
It would make sense. I'll change it.